src/embedded-checkout/iframe-content/embedded-content-event-handler.ts

```diff
@@ -0,0 +1,23 @@
+import { EmbeddedCheckoutEvent, EmbeddedCheckoutEventType, EmbeddedCheckoutFrameLoadedEvent } from '../embedded-checkout-events';
+
+export default class EmbeddedContentEventHandler {
+    handle(message: EmbeddedCheckoutEvent): void {
```

Comment:
I was wondering if instead of having a `EmbeddedContentEventHandler`, we should be able to register different handlers against different event types? 🍹